README.md

```diff
@@ -19,14 +19,14 @@ ES-Hadoop 2.0.x and 2.1.x are compatible with Elasticsearch __1.X__ *only*
 
 ## Installation
 
-### Stable Release (currently `7.7.0`)
+### Stable Release (currently `7.7.1`)
```

Comment:
```suggestion
### Stable Release (currently `7.7.0)
```

Since this is a new minor, the stable release number stays the same in the readme.